tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38, coverage, pylint
+envlist = py{38,39,310,311,312}, coverage, pylint
```

We are only using py38 and py312 in CI so we can skip rest of the versions.
```suggestion
envlist = py{38,312}, coverage, pylint
```